node-daemon/RPC_INTERFACE.md

```diff
@@ -0,0 +1,1170 @@
+# RPC documentation for Mintlayer node
+
+Version `0.3.0`.
+
+## Module `node`
+
+RPC methods controlling the node.
+
+
+### Method `node_shutdown`
+
+Order the node to shutdown
+
+
+Parameters:
+```
+{}
+```
+
+Returns:
+```
+nothing
+```
+
+
+### Method `node_version`
+
+Get node software version
+
+
+Parameters:
+```
+{}
+```
+
+Returns:
+```
+string
+```
+
+
+### Method `node_set_mock_time`
```

Comment:
Perhaps we should have a directive to hide some methods from documentation. But this isn't a priority. You can do this next/later, once we have something that fully works.